### PR TITLE
Testing platform new output parameters

### PR DESCRIPTION
**Pull request by nohwnd**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-platform-extensions-output.md](https://github.com/dotnet/docs/blob/dde4bf12051460e7d16d6071d199fe33572045fd/docs/core/testing/unit-testing-platform-extensions-output.md) | [docs/core/testing/unit-testing-platform-extensions-output](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-extensions-output?branch=pr-en-us-42328) |

### CODE REVIEW

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Microsoft.Testing.Platform policy extensions
+description: Learn about the various Microsoft.Testing.Platform policy extensions and how to use them.
+author: evangelink
+ms.author: amauryleve
+ms.date: 04/10/2024
```
Comment (nohwnd) [inline docs/core/testing/unit-testing-platform-extensions-output.md:6]: fixing

Comment (nohwnd) [reply docs/core/testing/unit-testing-platform-extensions-output.md:6]:
```suggestion
title: Microsoft.Testing.Platform output extensions
description: Learn about the various Microsoft.Testing.Platform output extensions and how to use them.
author: nohwnd
ms.author: jajares
ms.date: 08/26/2024
```

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# Output extensions
+
+This article list and explains all `Microsoft Testing Platform` extensions related to the terminal output.
+
+## Terminal test reporter
+
+Terminal test reporter is the default implementation of status and progress reporting to the terminal (console).
+
+It comes built-in with Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-extensions-output.md:17]:
```suggestion
It comes built-in with Microsoft.Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
```

Comment (Evangelink) [reply docs/core/testing/unit-testing-platform-extensions-output.md:17]: Actually I think there are too many "and", what about:

```suggestion
It comes built-in with `Microsoft.Testing.Platform`, offering both ANSI and non-ANSI modes along with a progress indicator.
```

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Output modes
+
+![Output with 1 failed test and a summary](./media/test-output-and-summary.png)
+There are two output modes available:
+
+- Normal, the output contains the banner, reports full failures of tests, warning messages and writes summary of the run.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-extensions-output.md:24]:
```suggestion
- `Normal`, the output contains the banner, reports full failures of tests, warning messages and writes summary of the run.
```

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- Detailed, the same as `Normal` but it also reports `Passed` tests.
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-extensions-output.md:25]:
```suggestion
- `Detailed`, the same as `Normal` but it also reports `Passed` tests.
```

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Terminal test reporter
+
+Terminal test reporter is the default implementation of status and progress reporting to the terminal (console).
+
+It comes built-in with Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
+
+### Output modes
+
+![Output with 1 failed test and a summary](./media/test-output-and-summary.png)
+There are two output modes available:
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-platform-extensions-output.md:22]: I think it'd be good to have a sample of code and see the screenshot for each mode.

Comment (nohwnd) [reply docs/core/testing/unit-testing-platform-extensions-output.md:21]: added

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+# Output extensions
+
+This article list and explains all `Microsoft Testing Platform` extensions related to the terminal output.
+
+## Terminal test reporter
+
+Terminal test reporter is the default implementation of status and progress reporting to the terminal (console).
+
+It comes built-in with Microsoft.Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-platform-extensions-output.md:17]:
```suggestion
It comes built-in with `Microsoft.Testing.Platform`, and offers ANSI and non-ANSI mode, and progress indicator.
```

File: docs/core/testing/unit-testing-platform-extensions-output.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+title: Microsoft.Testing.Platform output extensions
+description: Learn about the various Microsoft.Testing.Platform output extensions and how to use them.
+author: nohwnd
+ms.author: jajares
+ms.date: 08/26/2024
+---
+
+# Output extensions
+
+This article list and explains all `Microsoft Testing Platform` extensions related to the terminal output.
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-platform-extensions-output.md:11]:
```suggestion
This article list and explains all **Microsoft Testing Platform** extensions related to the terminal output.
```

**Review by IEvangelist — APPROVED** [review]

File: docs/core/testing/unit-testing-platform-extensions-output.md (hunk 6 above)
Comment (adegeo) [reply docs/core/testing/unit-testing-platform-extensions-output.md:17]: This one contradicts the style in the other one. I suggest using **Microsoft.Testing.Platform**.

Comment (nohwnd) [reply docs/core/testing/unit-testing-platform-extensions-output.md:17]:
```suggestion
It comes built-in with **Microsoft.Testing.Platform**, and offers ANSI and non-ANSI mode, and progress indicator.
```

Comment (nohwnd) [reply docs/core/testing/unit-testing-platform-extensions-output.md]: Used bold, as suggested.

**Pull request merged**